core/deployment/src/main/java/io/quarkus/deployment/execannotations/ExecutionAnnotationsConfig.java

```diff
@@ -0,0 +1,35 @@
+package io.quarkus.deployment.execannotations;
+
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+import io.smallrye.config.ConfigMapping;
+import io.smallrye.config.WithDefault;
+
+@ConfigRoot(phase = ConfigPhase.BUILD_TIME)
+@ConfigMapping(prefix = "quarkus.execution-annotations")
```

Comment:
`execution-model-annotations` maybe?

Reply:
Sounds fair. I'll update the other places to use "execution model annotations" or "annotations affecting the execution model" or so.